tests/conftest.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+
+import operator
+
+import pytest
+
+
+@pytest.fixture
+def dtype():
+    """A fixture providing the ExtensionDtype to validate."""
+    raise NotImplementedError
+
+
+@pytest.fixture
+def data():
```

Comment:
These fixtures are duplicated in `tests/string_array/test_pandas_extension.py`. Can we get rid of them here or is there a special reason to keep them?

Reply:
I copied it from pandas to define the interface. these fixtures need to be implemented by the subclasses. thought this would help readability

Reply:
I would expect that we will only implement these fixtures once and let them depend on another fixture that will make the switch between the  various supported types. Thus this file here is rather documentation-like stuff that should only reside in Pandas.